tests/buyProduct_test.js

```diff
@@ -0,0 +1,42 @@
+const USER = {
+  email: 'testtest123123@gmail.com',
+  password: 'Test',
+};
+
+const CHECKOUT_INFO = {
+  char: 'Test',
+  number: 1243,
+};
+Feature('purchase');
+
+Scenario.only('buy product', async ({ I, basePage }) => {
+  I.openStore();
+  I.login(USER);
+  I.addItemToCart();
+  const checkoutPrice = await I.grabTextFrom({ xpath: '//div/span[@class="price-cart"]' });
```

Comment:
You need at least two page objects: product(select products, extract prices, add to cart) and checkout(extract total price, shipping, complete purchase)